src/components/subject-homepage/subject-sidebar.tsx

```diff
@@ -91,9 +104,12 @@ const SidebarItem = ({ unit }: { unit: Unit }) => {
           {unit.chapters.map((chapter) => (
             <Link
               className="group mb-3 flex items-center gap-x-1.5 text-sm font-medium last:mb-0"
-              href={`${pathname}/${unit.title.toLowerCase().replace(/[^a-z1-9 ]+/g, "").replace(/\s/g, "-")}/${chapter.chapter}`}
+              href={`${pathname}/${unit.title
+                .toLowerCase()
+                .replace(/[^a-z1-9 ]+/g, "")
+                .replace(/\s/g, "-")}/${chapter.chapter}`}
```

Comment:
should use `replaceAll` instead of `replace /g` here

> The [String#replaceAll()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) method is both faster and safer as you don't have to use a regex and remember to escape it if the string is not a literal. And when used with a regex, it makes the intent clearer.

\- [source](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/77f32e5a6b2df542cf50dfbd371054f2cd8ce2d6/docs/rules/prefer-string-replace-all.md)